lib/presentation/core/widgets/authentication/custom_text_form_field.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+class CustomTextFormField extends StatelessWidget {
+  const CustomTextFormField({
+    Key? key,
+    required this.keyboardType,
+    required this.hintText,
+    this.prefixIcon,
+    this.suffixIcon,
+    this.obscureText = false,
+  }) : super(key: key);
+  final TextInputType keyboardType;
+  final String hintText;
+  final Widget? prefixIcon;
+  final Widget? suffixIcon;
+  final bool obscureText;
+  @override
+  Widget build(BuildContext context) {
+    return TextFormField(
+      obscureText: obscureText,
+      decoration: InputDecoration(
+          suffixIcon: suffixIcon,
+          prefixIcon: prefixIcon,
+          hintText: hintText,
+          filled: true,
+          border: OutlineInputBorder(
+            borderRadius: BorderRadius.circular(15),
```

Comment:
In file `lib/presentation/core/widgets/authentication/form_submit_buton.dart` you are also using the same Radius. Consider extracting "magic number" - 15 to some constant, e.g. `FormCircularRadiusValue` to make sure future forms will be consistent.

You can read more about magic numbers <a href="https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad"> here </a>